src/routes/LostItemsContainer.js

```diff
@@ -1,23 +1,196 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
 
-import { withStripes } from '@folio/stripes/core';
+import {
+  stripesConnect,
+} from '@folio/stripes/core';
+import {
+  makeQueryFunction,
+  StripesConnectedSource,
+  parseFilters,
+  buildUrl,
+} from '@folio/stripes/smart-components';
 
-import NoPermissionMessage from '../components/NoPermissionMessage/NoPermissionMessage';
+import {
+  LostItemsListContainer,
+  NoPermissionMessage,
+} from '../views/LostItems';
+import {
+  ACTUAL_COST_RECORD_FIELD_NAME,
+  ACTUAL_COST_RECORD_FIELD_PATH,
+  PAGE_AMOUNT,
+  SEARCH_FIELDS,
+} from '../views/LostItems/constants';
 
-const LostItemsContainer = ({ stripes }) => {
-  const hasPermission = stripes.hasPerm('ui-users.lost-items.requiring-actual-cost');
+function buildFilterConfig(filters) {
+  const customFilterConfig = [];
+  const parsedFilters = parseFilters(filters);
 
-  return (
-    !hasPermission && <NoPermissionMessage content={<FormattedMessage id="ui-users.lostItems.message.noAccessToActualCostPage" />} />
-  );
-};
+  Object.keys(parsedFilters).forEach(name => {
+    if (name.match('customFields')) {
+      customFilterConfig.push(
+        {
+          name,
+          cql: name.split('-').join('.'),
+          values: [],
+          operator: '=',
+        },
+      );
+    }
+  });
 
-LostItemsContainer.propTypes = {
-  stripes: PropTypes.shape({
-    hasPerm: PropTypes.func.isRequired,
-  }).isRequired,
-};
+  return customFilterConfig;
+}
 
-export default withStripes(LostItemsContainer);
+const filterConfig = [
+  {
+    name: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.LOSS_TYPE],
+    cql: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.LOSS_TYPE],
+    values: [],
+  },
+];
+
+function buildQuery(queryParams, pathComponents, resourceData, logger, props) {
+  const customFilterConfig = buildFilterConfig(queryParams.filters);
+
+  return makeQueryFunction(
+    'cql.allRecords=1',
+    SEARCH_FIELDS.map(index => `${index}="%{query.query}*"`).join(' or '),
+    {},
+    [...filterConfig, ...customFilterConfig],
+    2,
+  )(queryParams, pathComponents, resourceData, logger, props);
+}
+
+class UserSearchContainer extends React.Component {
```

Comment:
As for me it is a very strange name. 
Could you use something like `LostItemsContainer` or something similar?

Reply:
Fixed